pynecone/app.py

```diff
@@ -85,24 +100,6 @@ def __call__(self) -> FastAPI:
         """
         return self.api
 
-    def add_default_endpoints(self):
-        """Add the default endpoints."""
-        # To test the server.
-        self.api.get(str(constants.Endpoint.PING))(ping)
-
-        # To make state changes.
-        self.api.websocket(str(constants.Endpoint.EVENT))(event(app=self))
-
-    def add_cors(self):
```

Comment:
Do we not need this anymore?

Reply:
It's taken care of when the `AsyncServer` is created on the `App` `__init__` :

        self.sio = AsyncServer(async_mode="asgi", cors_allowed_origins="*")

There's also `cors_credentials=` which defaults to `True` already. 

python-scoketio does not have CORS options like `allow_methods` and `allow_headers` but I believe it's configured to avoid cross-origin problems with browsers by allowing all headers and methods anyway.

Reply:
Okay got it nice